src/webgpu/gpu_test.ts

```diff
@@ -544,6 +544,15 @@ export class GPUTestBase extends Fixture<GPUTestSubcaseBatchState> {
     }
   }
 
+  skipIfMultisampleNotSupportedForFormat(...formats: (GPUTextureFormat | undefined)[]) {
+    for (const format of formats) {
+      if (format === undefined) continue;
+      if (!isMultisampledTextureFormat(format, this.isCompatibility)) {
```

Comment:
I think if rebase you'll see this has changed.

```suggestion
      if (!isTextureFormatMultisampled(device, format)) {
```

And you'll need to import it at the top. Sorry for the conflict.

Reply:
GitHub is showing a merge conflict now for me. Probably it took a while to refresh the mergability status.

Reply:
Sorry I double stomped on this as I needed this for a PR. The function is

https://github.com/gpuweb/cts/blob/a6e2c34fee3b56f79df5fc1e66d990a0734ccd3c/src/webgpu/gpu_test.ts#L650

I've been trying to move `TextureFormat` to the beginning so typing `skipIfT` brings up the completions related to textures.